<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>QgsVectorLayerPropertiesBase</class>
 <widget class="QDialog" name="QgsVectorLayerPropertiesBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Layer Properties</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="5" >
    <widget class="QTabWidget" name="tabWidget2" >
     <property name="tabShape" >
      <enum>QTabWidget::Rounded</enum>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Symbology</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="1" column="1" >
        <widget class="QSlider" name="sliderTransparency" >
         <property name="maximum" >
          <number>255</number>
         </property>
         <property name="value" >
          <number>0</number>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="lblTransparencyPercent" >
         <property name="text" >
          <string>Transparency:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2" >
        <widget class="Q3WidgetStack" name="widgetStackRenderers" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>7</hsizetype>
           <vsizetype>7</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <widget class="QWidget" name="qt_dead_widget_singlesympage" >
          <layout class="QGridLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
          </layout>
         </widget>
         <widget class="QWidget" name="qt_dead_widget_continuoussympage" >
          <layout class="QGridLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
          </layout>
         </widget>
         <widget class="QWidget" name="qt_dead_widget_WStackPage" />
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="legendtypelabel" >
         <property name="maximumSize" >
          <size>
           <width>32767</width>
           <height>22</height>
          </size>
         </property>
         <property name="text" >
          <string>Legend type:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QComboBox" name="legendtypecombobox" >
         <property name="minimumSize" >
          <size>
           <width>270</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>32767</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>8</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="2" column="0" colspan="4" >
        <widget class="QGroupBox" name="chkUseScaleDependentRendering" >
         <property name="title" >
          <string>Use scale dependent rendering</string>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>8</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="0" column="2" >
           <widget class="QLabel" name="textLabel1_2_2" >
            <property name="text" >
             <string>Maximum 1:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="textLabel1" >
            <property name="text" >
             <string>Minimum 1:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QSpinBox" name="spinMinimumScale" >
            <property name="toolTip" >
             <string>Minimum scale at which this layer will be displayed. </string>
            </property>
            <property name="maximum" >
             <number>100000000</number>
            </property>
            <property name="minimum" >
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="0" column="3" >
           <widget class="QSpinBox" name="spinMaximumScale" >
            <property name="toolTip" >
             <string>Maximum scale at which this layer will be displayed. </string>
            </property>
            <property name="maximum" >
             <number>100000000</number>
            </property>
            <property name="minimum" >
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="textLabel3" >
         <property name="text" >
          <string>Display name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3" >
        <widget class="QComboBox" name="displayFieldComboBox" >
         <property name="whatsThis" >
          <string>Use this control to set which field is placed at the top level of the Identify Results dialog box.</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="4" >
        <widget class="QGroupBox" name="grpSRS" >
         <property name="title" >
          <string>Spatial Reference System</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>8</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="0" column="0" >
           <widget class="QLineEdit" name="leSpatialRefSys" >
            <property name="readOnly" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QPushButton" name="pbnChangeSpatialRefSys" >
            <property name="text" >
             <string>Change</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="2" >
        <widget class="QLabel" name="textLabel2" >
         <property name="toolTip" >
          <string>Display field for the Identify Results dialog box</string>
         </property>
         <property name="whatsThis" >
          <string>This sets the display field for the Identify Results dialog box</string>
         </property>
         <property name="text" >
          <string>Display field</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3" >
        <widget class="QLabel" name="lblSource" >
         <property name="maximumSize" >
          <size>
           <width>32767</width>
           <height>22</height>
          </size>
         </property>
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text" >
          <string>TextLabel2</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="4" >
        <widget class="QGroupBox" name="grpSubset" >
         <property name="title" >
          <string>Subset</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>8</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="0" column="0" colspan="2" >
           <widget class="Q3TextEdit" name="txtSubsetSQL" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QPushButton" name="pbnQueryBuilder" >
            <property name="text" >
             <string>Query Builder</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>480</width>
              <height>21</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="textLabel3_2" >
         <property name="maximumSize" >
          <size>
           <width>32767</width>
           <height>22</height>
          </size>
         </property>
         <property name="text" >
          <string>Layer source</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="4" >
        <widget class="QGroupBox" name="indexGroupBox" >
         <property name="title" >
          <string>Spatial Index</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>8</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="0" column="0" >
           <widget class="QLabel" name="indexLabel" >
            <property name="text" >
             <string>Create Spatial Index</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" >
           <widget class="QPushButton" name="pbnIndex" >
            <property name="text" >
             <string>Create</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>141</width>
              <height>21</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="txtDisplayName" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Metadata</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>8</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="Q3TextEdit" name="teMetadata" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Labels</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>8</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="1" column="0" >
        <widget class="Q3Frame" name="labelOptionsFrame" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>7</hsizetype>
           <vsizetype>7</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape" >
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Plain</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QCheckBox" name="labelCheckBox" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>7</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Display labels</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TabPage" >
      <attribute name="title" >
       <string>Actions</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>8</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="Q3Frame" name="actionOptionsFrame" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>7</hsizetype>
           <vsizetype>7</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape" >
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QPushButton" name="btnHelp" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Help</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4" >
    <widget class="QPushButton" name="pbnCancel" >
     <property name="text" >
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <widget class="QPushButton" name="pbnOK" >
     <property name="text" >
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QPushButton" name="pbnApply" >
     <property name="text" >
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>270</width>
       <height>29</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3TextEdit</class>
   <extends></extends>
   <header>q3textedit.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>Q3WidgetStack</class>
   <extends></extends>
   <header>q3widgetstack.h</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>Q3Frame</class>
   <extends></extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
